### PR TITLE
Check that GCC used to compile has a bigger version than 4.8.0

### PR DESCRIPTION
**Pull request by JelteF**
This is a check to ensure that Citus Enterprise is not built with a GCC version
lower than 4.8.0. This is mostly used as a way to ensure that el/6 and ol/6 are
not accidentily built before #424 is merged.

### CODE REVIEW

File: citus-enterprise.spec
Comment (hanefi) [inline citus-enterprise.spec:48]: I am curious, why do we need `CC=$(command -v gcc)` here?

**Review by hanefi — APPROVED** [review]
LGTM, just one question out of curiosity. I am sure it has a valid and logical answer

**Review by hanefi — CHANGES_REQUESTED** [review]
I just realized failed builds on `EL/6`

LGTM after we fix the issues there. Blocked until https://github.com/citusdata/packaging/pull/424 is merged I guess

File: citus-enterprise.spec
Comment (JelteF) [reply citus-enterprise.spec:48]: Mainly to be sure from the build output that we are using the right one. I don't think it's really needed per se, because `pg_config --cc` outputs `gcc --std=gnu99`. If it would use an absolute path though it would be an issue.

**Pull request merged**
